app/views/public_revisions/new_public_revision.html.haml

```diff
@@ -6,5 +6,7 @@
   = f.input :pages, label „Number of pages”, hint „Ilość stron”
   = f.input :pictures, label „Number of pictures”, hint „Ilość obrazków”
   = f.input :received, label: "Otrzymano", class: %w{datepicker form-control}, as: :string, input_html: { value: l(f.object.received, format: "%d-%m-%Y"), type:"date" } 
+  = f.text_area :reply_to_the_1reviewer, placeholder: "Reply to the first reviewer"
```

Comment:
A jeśli będzie więcej recenzentów? Trzeba to wygenerować na podstawie liczby recenzji przypisanych do tego zgłoszenia.